src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/WfsLayer.tsx

```diff
@@ -14,6 +14,7 @@ import type { DataLayerProps } from '../../types'
 import { NO_DATA, WfsDataProvider } from './context'
 
 const SRS_NAME = 'urn:ogc:def:crs:EPSG::4326'
+const SRS_NAME_SMALL = 'EPSG:4326'
```

Comment:
What makes this differentiation necessary?

Reply:
As per the url shown in the description above, only this value seems to work, not the long one.

Reply:
To me, the name `srsNameSmall` doesn't convey any meaning; without documentation it is not clear what the difference is and why its introduction is required. I would strongly suggest looking into using the current `wfsFilter` value. The WFS standard offers a lot of filter options. There has to be an option that allows us to query GeoServer services without having to add parameters for each and every exception.

Reply:
I agree. For me WFS services are just always very difficult to get working. Maybe it's this specific service of Haarlem not allowing everything. I'll discuss with them.

Reply:
How about this. One SRS value that seems to work everywhere.

Reply:
Looks good. Seems to work as well 😄